classes/02class/exercises/c02-network03/afaryy/README.md

```diff
@@ -0,0 +1,63 @@
+# c02-network03
+
+## Commands Execution Output
+
+- Commands for creating a new VPC:
+```CLI
+$VPC_ID=$(aws ec2 create-vpc \
+    --cidr-block 10.128.0.0/16 \
+    --no-amazon-provided-ipv6-cidr-block \
+    --instance-tenancy default \
+    --tag-specifications 'ResourceType=vpc,Tags=[{Key=Name,Value=da-c02-vpc}]' \
+    --query Vpc.VpcId \
+    --output text)
+
+$aws ec2 create-tags --resources $VPC_ID --tags Key=Env,Value=test
```

Comment:
Nice, now you have the `--tag-specification` you dont need this.

Or, if you wanted to add more tags you could use `Tags=[{Key=Name,Value=da-c02-vpc},{Key=Env,Value=Test}]`

But nice to have both examples in your answer